File: Daytwo/python/task_six.py
def sum_element_of_even_index(arrayPassedIn):
	evenIndex = [];
	sum = 0;
	int counter = 0;
	for number in range 10:
		if(number%2==0):
			evenIndex[counter] = arrayPassedIn[number]
			sum+=evenIndex[counter]
			counter+=1
	return sum

File: Daytwo/java/TaskSix.java
public class TaskSix{
	public static int elementOfEvenIndex(int[] arrayPassedIn){
		int sum = 0;
		for(int count = 0; count<10;count++){
			if(count%2==0){
				sum += arrayPassedIn[count];
			}
		}
	return sum;
	}
}